server/src/index.ts

```diff
@@ -398,9 +468,17 @@ app.get(
   },
 );
 
-app.get("/api/loadPromotedContentGroups", (req: Request, res: Response) => {
-  res.send({});
-});
+app.get(
+  "/api/loadPromotedContentGroups",
+  async (req: Request, res: Response) => {
+    const groups = await loadPromotedContentGroups();
+    console.log(groups);
+    res.send({
+      success: true, //TODO: do we need this?
```

Comment:
Don't send `success`. That is determined by the http code. If the client gets a 200, it know is is a success.